packages/terra-i18n-plugin/src/I18nAggregatorPlugin.js

```diff
@@ -15,42 +14,48 @@ const messages = ${JSON.stringify(messages, null, 2)};
 module.exports = {
   areTranslationsLoaded: true,
   locale: '${language}',
-  messages,
+  messages: messages
 };`;
 }
 
-function getDirectories(srcPath) {
-  return fs.readdirSync(srcPath).filter(file => fs.statSync(path.join(srcPath, file)).isDirectory());
+function getDirectories(srcPath, compiler) {
+  return compiler.inputFileSystem.readdirSync(srcPath).filter(file => compiler.inputFileSystem.statSync(path.join(srcPath, file)).isDirectory());
 }
 
-function aggregateDirectory(languageMessages, currentDirectory) {
+function aggregateDirectory(languageMessages, currentDirectory, compiler) {
   // Check the directory for translations
   const translationsDirectory = path.resolve(currentDirectory, 'translations');
-  if (fs.existsSync(translationsDirectory)) {
+  try {
+    // Check if the directory exists by attempting to read from it
+    compiler.inputFileSystem.readdirSync(translationsDirectory);
+
     // Check the directory for each translation file
     supportedLocales.forEach((language) => {
       const translationFile = path.resolve(translationsDirectory, `${language}.json`);
-      if (fs.existsSync(translationFile)) {
-        Object.assign(languageMessages[language], JSON.parse(fs.readFileSync(translationFile, 'utf8')));
-      } else {
-        // eslint-disable-next-line no-console
+      try {
+        Object.assign(languageMessages[language], JSON.parse(compiler.inputFileSystem.readFileSync(translationFile, 'utf8')));
+      } catch (e) {
         console.warn(`Translation file ${language}.json not found for ${translationsDirectory}`);
       }
     });
+  } catch (e) {
+    // eslint-disable-line no-use-before-defin
   }
 
   // Check the directory's node_modules for translation files
   const nodeMoudlesPath = path.resolve(currentDirectory, 'node_modules');
-  if (fs.existsSync(nodeMoudlesPath)) {
```

Comment:
We should probably keep this. If I remember correctly, the `.bin` directory is an example of a node_modules dir that does not contain a node_modules itself which was the original reason for this check.

Reply:
I wonder if this is the reason for the excessive console logs you were seeing.

Reply:
The Try catch is replacing the exists call because the default inputfile system provided by node does not implement existsSync. This is also why we aren't outputting information in the catch blocks, because it's not really an error that the file doesn't exist.

Reply:
The goal of this is to remove as much of the fs as possible and use the compilers inputFileSystem; hence, no existsSynch exists anymore.  Therefore, we use the try catch.  We attempt to read from the directory instead inside getDirectories, which will throw the exception that gets caught.